src/geometryConcrete/GeometryConcrete.cpp

```diff
@@ -49,21 +49,7 @@ void NuTo::GeometryConcrete::MaximizeParticleDistance(double rParticleDistance)
 
     double timeEnd = .5*rParticleDistance/mAbsoluteGrowthRate;
 
-    try
-    {
-        handler.Simulate(mNumEventsMax, timeEnd, mSecondsWallTimeMax, mSecondsPrint, mInitialTimeBarrier);
-    } catch (NuTo::Exception& e)
-    {
-        e.AddMessage("The simulation stopped with an exception. \n");
-        if (mContinueOnException)
-        {
-            std::cout << e.ErrorMessage() << "\n but I'll continue.";
-        }
-        else
-        {
-            throw;
-        }
-    }
+    handler.Simulate(mNumEventsMax, timeEnd, mSecondsWallTimeMax, mSecondsPrint, mInitialTimeBarrier);
```

Comment:
!!! You changed the logic flow. `if (mContinueOnException)`, the code does __not rethrow__.